api/tests/conftest.py

```diff
@@ -107,7 +107,8 @@ def make_db_record(job_id,
 
 
 def setup_requests_mock(batch):
-    granules = [job['job_parameters']['granule'] for job in batch]
+    granules = []
+    [[granules.append(granule) for granule in job['job_parameters']['granules']] for job in batch]
```

Comment:
this is a list comprehension, but we don't assign it to a variable?  odd way to implement a loop, I think it can be:
`granules = [granule for granule in job['job_parameters']['granules'] for job in batch]`